examples/simple_adding.rb

```diff
@@ -0,0 +1,11 @@
+def simple_adding(number)
+  i = 0
+  sum = 0
+  number.times do
```

Comment:
`.times` comes with a built in counter 😃. Name it whatever you want. I've named it `counter`. To make it an easy refactor, just use the name `i`. 

```ruby
number.times do |counter|
  # your code
end
```